src/Plugin/Field/FieldFormatter/MediaImageFormatter.php

```diff
@@ -33,6 +33,10 @@ protected function getStyleOptions() {
   public function preRender($element) {
     $source_field = self::getSourceField($element['#media']);
 
+    if (empty($element[$source_field]['#field_type']) || $element[$source_field]['#field_type'] != 'image') {
```

Comment:
Are non-images getting to this through multimedia or a mis-configuration in Drupal?

Reply:
if a media field allows for image & videos. Its possible but we aren't doing it at the moment